{"name": "Delete pet once again", "status": "failed", "statusDetails": {"message": "AssertionError: assert 200 == 404\n +  where 200 = <Response [200]>.status_code", "trace": "self = <test_api.TestApi object at 0x000001876502C1D0>, api_client = <utils.api_client.ApiClient object at 0x0000018764D29690>\n\n    @allure.title(\"Delete pet once again\")\n    def test_delete_pet_404(self, api_client):\n        response = api_client.delete(\"/pet/1\")\n        print(response)\n        with allure.step(f\"Check status {response.status_code}\"):\n>           assert response.status_code == 404\nE           assert 200 == 404\nE            +  where 200 = <Response [200]>.status_code\n\ntests\\test_api.py:193: AssertionError"}, "steps": [{"name": "Delete request to '/pet/1'", "status": "passed", "parameters": [{"name": "path", "value": "'/pet/1'"}], "start": 1704364893394, "stop": 1704364894513}, {"name": "Check status 200", "status": "failed", "statusDetails": {"message": "AssertionError: assert 200 == 404\n +  where 200 = <Response [200]>.status_code\n", "trace": "  File \"D:\\Pycharm_Projects\\Pytest_Exam\\tests\\test_api.py\", line 193, in test_delete_pet_404\n    assert response.status_code == 404\n"}, "start": 1704364894514, "stop": 1704364894514}], "attachments": [{"name": "stdout", "source": "2487e096-42af-4f65-af73-f6d2421442c5-attachment.txt", "type": "text/plain"}], "start": 1704364893393, "stop": 1704364894515, "uuid": "e9f4747a-d3d9-4aa9-b71b-fd1519116979", "historyId": "b781c4dae5e7f7ac2beda12a67c8462c", "testCaseId": "b781c4dae5e7f7ac2beda12a67c8462c", "fullName": "tests.test_api.TestApi#test_delete_pet_404", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "DESKTOP-323VRMN"}, {"name": "thread", "value": "9404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}